docs/units/equivalencies.rst

```diff
@@ -626,27 +626,27 @@ However, when passing the spectral equivalency, you can see there are
 all kinds of things that ``Hz`` can be converted to::
 
   >>> u.Hz.find_equivalent_units(equivalencies=u.spectral())
-    Primary name | Unit definition        | Aliases
+  Primary name | Unit definition        | Aliases
   [
-    AU           | 1.49598e+11 m          | au, astronomical_unit ,
-    Angstrom     | 1e-10 m                | AA, angstrom          ,
-    Bq           | 1 / s                  | becquerel             ,
-    Ci           | 3.7e+10 / s            | curie                 ,
-    Hz           | 1 / s                  | Hertz, hertz          ,
-    J            | kg m2 / s2             | Joule, joule          ,
-    Ry           | 2.17987e-18 kg m2 / s2 | rydberg               ,
-    cm           | 0.01 m                 | centimeter            ,
-    eV           | 1.60218e-19 kg m2 / s2 | electronvolt          ,
-    earthRad     | 6.3781e+06 m           | R_earth, Rearth       ,
-    erg          | 1e-07 kg m2 / s2       |                       ,
+    AU           | 1.49598e+11 m          | au, astronomical_unit            ,
```

Comment:
?

Reply:
I just copy & pasted this. I think the main change is that this expands it to encompass the long jupter radius entry.